src/ddmd/dinterpret.d

```diff
@@ -415,6 +415,12 @@ public:
             ctfeCompile(s.statement);
     }
 
+    override void visit(ForwardingStatement s)
+    {
+        assert(!!s.statement);
```

Comment:
assert(s.statement) is special syntax to invoke the class invariant. I always use either `!!` or `!is null` when asserting on a class reference.